compiler/src/dmd/backend/codebuilder.d

```diff
@@ -347,7 +347,7 @@ extern (C++) struct CodeBuilder
      * Generate code to deal with floatreg.
      */
     @trusted
-    void genfltreg(opcode_t opcode,uint reg,targ_size_t offset)
+    void genfltreg(opcode_t opcode,int reg,targ_size_t offset)
```

Comment:
Why make `reg` signed?

Reply:
`modregxrm` takes an `uint`

Reply:
The problem is, it isn't really a `reg_t` and can have negative values. I plan to fix it in a followon PR.